test/sagemaker_tests/tensorflow/inference/test/integration/local/test_multi_model_endpoint.py

```diff
@@ -205,7 +211,7 @@ def test_load_non_existing_model():
 
 
 @pytest.mark.skip_gpu
-def test_bad_model_reqeust():
+def test_bad_model_request():
```

Comment:
nit: pytest.mark.model("bad_model")

Reply:
I think this is more like non_existing_model (which is used elsewhere), but thanks for the callout

Reply:
i think it's a "bad model-request" rather than a "bad-model request"

Reply:
Ah, right. I was under the impression that it tests a model that is mangled somehow.